src/Directory.Build.targets

```diff
@@ -15,4 +15,28 @@
     <DefineConstants Condition=" $(IsOSXBuild) == 'true' ">$(DefineConstants);OSX_BUILD</DefineConstants>
     <DefineConstants Condition=" $(IsWindowsBuild) == 'true' ">$(DefineConstants);WINDOWS_BUILD</DefineConstants>
   </PropertyGroup>
+
+  <!-- Nuget Publishing related properties that are common to all packages -->
+  <PropertyGroup Condition="'$(EnableNuget)' == 'true'" >
```

Comment:
I can't use `GeneratePackageOnBuild` here because I have to inject signing before the `Pack` target runs.